tests/Label/ReadModels/JSON/Repository/Doctrine/BaseDBALRepositoryTest.php

```diff
@@ -39,7 +40,13 @@ protected function saveEntity(Entity $entity): void
 
         $sql = 'INSERT INTO ' . $this->tableName . ' VALUES (?, ?, ?, ?, ?)';
 
-        $this->connection->executeQuery($sql, $values);
+        $this->connection->executeQuery($sql, $values, [
+            PDO::PARAM_STR,
```

Comment:
Do we need to use the built-in `PDO` types here or the `Doctrine\DBAL\Types\Types`?